model_analyzer/analyzer.py

```diff
@@ -119,9 +119,17 @@ def profile(self, client, gpus):
         profiled_model_list = list(
             self._state_manager.get_state_variable(
                 'ResultManager.results').keys())
+
+        num_profiled_configs = sum([
```

Comment:
I know the existing code isn't clean, but lets make sure our new code is clean.  I asked below to use a new logger line for this, so you should be able to extract it all out into a function that is called after the normal "finished profiling" print.  Same issue down below for the other case.  Once you have standalone functions you can add unit tests.

Reply:
extracted both into separate functions and added unit tests 👍